sql/sql_parse.cc

```diff
@@ -2572,7 +2572,7 @@ static bool lock_tables_for_backup(THD *thd)
   @return FALSE in case of success, TRUE in case of error.
 */
 
-static bool lock_binlog_for_backup(THD *thd)
+bool lock_binlog_for_backup(THD *thd)
```

Comment:
That's right, I missed it in #1270. Yes, we would need a blueprint "expose LOCK BINLOG FOR BACKUP function to plugins", but it's borderline silly in this instance. I'm willing to let this one slide